src/i18n/en-us/index.js

```diff
@@ -116,6 +118,11 @@ export default {
         rows_per_page: 'Rows per page',
         amount: 'Amount',
     },
+    nft : {
+        link_to_nft_details: 'Go to NFT details page for {name}',
+        img_alt: 'NFT - {nftInfo}',
+        img_alt_video_nft: 'Video NFT cover - {nftInfo}',
+    },
```

Comment:
```suggestion
    nft : {
        link_to_nft_details: 'Go to NFT details page for {name}',
        img_alt: 'NFT - {nftInfo}',
        img_alt_video_nft: 'Video NFT cover - {nftInfo}',
        details_page_title: 'NFT Details Page',
        tile_toggle_radio_label: 'Show as tile?'
    },
```

Reply:
this will need to be removed in the future because this text will not actually be on the page btw